workbench-core/example/infrastructure/integration-tests/support/resources/dynamicAuthorization/groups.ts

```diff
@@ -40,6 +40,10 @@ export default class Groups extends CollectionResource {
     return this._axiosInstance.get(`${this._api}/users/${userId}`);
   }
 
+  public async getGroupUsers(groupId: string): Promise<AxiosResponse> {
+    return this._axiosInstance.get(`${this._api}/${groupId}/getUsers`);
+  }
+
```

Comment:
this function should go in the `Group` class since its called upon an individual group, not the groups collection

Reply:
Fixed !